server/graph/schemas/system.graphql

```diff
@@ -66,32 +66,32 @@ input SystemFlagInput {
 type SystemInfo {
   configFile: String @auth(requires: ["manage:system"])
   cpuCores: Int @auth(requires: ["manage:system"])
-  currentVersion: String @auth(requires: ["manage:system"])
+  currentVersion: String @auth(requires: ["manage:navigation"])
```

Comment:
IMO, user without manage:system permission should not see the currentVersion of wiki.js, he/she should not be allowed to know this, as they may leak this information accidentally.